### ISSUE TITLE
Issues with CanvasBlocker + YouTubeCenter

### ISSUE THREAD

**Issue by VADemon** [kkapsner/CanvasBlocker#12]
CanvasBlocker seems to interfere with YouTubeCenter plugin.

While CanvasBlocker is working, YTC will only work right after installation (only once!) and fail on following page reloadings, Firefox restarts etc.

YTC started working again after I disabled CanvasBlocker.
https://github.com/YePpHa/YouTubeCenter

CanvasBlocker settings:
Whitelist: `kkapsner.de,^https?://(\w+\.)*google.[a-z]+/maps,orteil.dashnet.org/cookieclicker/,youtube.com,steamcommunity.com`
Blacklist is empty
Blockmode: "Ask for permission for invisible canvas"
Canvas in PDFs is allowed

**Comment by kkapsner** [comment]
Duplicate of #7

But your Whitelist suggests you are not using the latest version. Which one are use using?
